redis/redis-cmd.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+#
+# Set the ACL of the redis instance
+# Strip any rights from the redis default user.
+echo "user default off -@all" > /tmp/redis.conf
+REDIS_PASSWORD="${REDIS_PASSWORD:+>$REDIS_PASSWORD}"
+
+## Add only a selection of user rights, if no user was passed via env use the default user same for password
+echo "user ${REDIS_USERNAME:-default} on +@all -@admin -@dangerous +flushdb ~* &* ${REDIS_PASSWORD:-nopass}" >> /tmp/redis.conf
```

Comment:
You can still use it without ACL's and TLS in dev mode. 
The reason why I introduced that was the following, since redis is the caching system for the zarr stuff and since in theroy you could create a distributed system that is also able to load data from say JSC this redis connection HAS to be made as secure as possible.

Reply:
I was referring to

> -@admin -@dangerous

What I typically use in redis-cli is the `KEYS` command to look into all available keys, which is apparently either `dangerous` or `admin`, I don't know. What I understand from this line is that you are stripping those permissions in any case, or have I gotten it wrong?

Reply:
Ahh, I see. Ok since the connection should be fairly saved username + passwd + ssl we can add admin and dangerous commands.